utils/license_chg.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+"""
+   (C) Copyright 2020 Intel Corporation.
+
+   SPDX-License-Identifier: BSD-2-Clause-Patent
+
+   Python tool to replace license text in various source code files.  Delete this
+   file once license change is complete.
+"""
+
+import argparse, pathlib
+
+
+if __name__ == "__main__" :
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('file', type=pathlib.Path)
+    args = parser.parse_args()
+    its_go = False
+
+    if str(args.file).endswith('.c') or str(args.file).endswith('.h'):
+        new_license = "* SPDX-License-Identifier: BSD-2-Clause-Patent\n"
+        old_license = "* Licensed under the Apache"
+        end_license_section = " */"
+    elif str(args.file).endswith('.py'):
+        new_license = "SPDX-License-Identifier: BSD-2-Clause-Patent\n"
+        old_license = "Licensed under the Apache"
+        end_license_section = tuple(('"""', "'''"))
```

Comment:
(pylint-redefined-variable-type) Redefinition of end_license_section type from str to tuple